build/args/all.gn

```diff
@@ -15,14 +15,7 @@ v8_enable_javascript_promise_hooks = true
 
 enable_cdm_host_verification = false
 ffmpeg_branding = "Chrome"
-
-# `rtc_use_h264` may be guarded behind is_chrome_branded alongside
-# proprietary_codecs https://webrtc-review.googlesource.com/c/src/+/36321,
-# explicitly override here with the same value as `proprietary_codecs`
-# to build OpenH264 encoder/FFmpeg decoder. The initialization of the
-# decoder depends on whether ffmpeg has been built with H.264 support.
-proprietary_codecs = false
-rtc_use_h264 = false
+proprietary_codecs = true
```

Comment:
I understand the part of this commit about removing `rtc_use_h264`, but was changing the value of `proprietary_codecs` in `all.gn` from `false` to `true` intentional? This seems like a big change?

Reply:
This is the current value which is required for h264/h265 mp4 video playback.
https://github.com/electron/electron/blob/26d228ccfe009400a06d0271ca84a7b08b574c1b/build/args/all.gn#L17

It's only disabled in the optionally included ffmpeg build
https://github.com/electron/electron/blob/26d228ccfe009400a06d0271ca84a7b08b574c1b/build/args/ffmpeg.gn#L5

Reply:
I believe it was mistakenly flipped in 30dad021336889faa6d71774b16ccc4ba3b6bb75

Reply:
You are right, 30dad02 is to blame! Let's not look too closely on who made that commit...